icepyx/core/query.py

```diff
@@ -797,6 +817,14 @@ def product_summary_info(self):
         ]
         for key in summ_keys:
             print(key, ": ", self._about_product["feed"]["entry"][-1][key])
+            
+    def latest_version(self):
+        """
+        A reference function to is2ref.latest_version.
+        
+        Determine the most recent version available for the given product.
+        """
```

Comment:
```suggestion
        Determine the most recent version available for the given product.
        
        Examples
        --------
        >>> reg_a = ipx.Query('ATL06',[-55, 68, -48, 71],['2019-02-20','2019-02-28'])
        >>> reg_a.latest_version()
        '006'
        """
```
Figured since we already had an example there we might as well use it!